src/raster/r.hydro.flatten/r.hydro.flatten.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+#
+#########################################################################
+#
+# MODULE:     r.hydro.flatten
+#
+# AUTHOR(S):  Anna Petrasova <kratochanna gmail com>
+#
+# PURPOSE:    Derive elevation of water bodies for hydro-flattening
+#
+# COPYRIGHT:  (C) 2023 by Anna Petrasova, and the GRASS Development Team
+#
+#             This program is free software under the GNU General Public
+#             License (>=v2). Read the COPYING file that comes with GRASS
+#             for details.
+#
+#########################################################################
+
+# %module
+# % description: Derive elevation of water bodies for hydro-flattening
+# % keyword: raster
+# % keyword: elevation
+# % keyword: hydrology
+# % keyword: lidar
+# % keyword: LIDAR
```

Comment:
Side note: we should switch to just one in code and addons, ideally lidar.

Reply:
What about LiDAR? (just kidding).